host_core/test/host_core/wasmcloud/native_test.exs

```diff
@@ -16,21 +16,9 @@ defmodule HostCore.WasmCloud.NativeTest do
 
     assert par.claims.public_key == @httpserver_key
     assert par.claims.issuer == @official_issuer
-    assert par.claims.version == "0.14.3"
+    assert par.claims.version == "0.14.4"
 
-    target_bytes =
-      case :os.type() do
-        {:unix, :darwin} ->
-          8_669_384
-
-        {:unix, _linux} ->
-          13_572_128
-
-        {:win32, :nt} ->
-          22_161_225
-      end
-
-    assert byte_size(par.target_bytes |> IO.iodata_to_binary()) == target_bytes
+    assert byte_size(par.target_bytes |> IO.iodata_to_binary()) > 8_000_000
```

Comment:
We may want to change this later but I'm OK with it for now. Feels weird to essentially assert a minimum provider executable size.

Reply:
Yeah I'm not really asserting a minimum provider executable size.. It just felt SUPER brittle asserting the file size of the provider every time. We know that the `oci-distribution` crate that we run on top of works properly, so I wanted to strip out some of the redundant and brittle-inducing assertions.

If we get a file size greater than 8MB, then we can be _reasonably sure_ that we got the right file from the OCI registry.

Reply:
Yeah agreed, it's nice to not have tests that _have_ to fail when we update. Perhaps when we get a more robust `wash par inspect` we can make actual assertions on size 😉